mm2src/mm2_main/src/wasm_tests.rs

```diff
@@ -164,10 +186,10 @@ async fn trade_base_rel_electrum(
 async fn trade_test_rick_and_morty() {
     let bob_policy = Mm2InitPrivKeyPolicy::Iguana;
     let alice_policy = Mm2InitPrivKeyPolicy::GlobalHDAccount;
-    let alice_path_to_address = StandardHDCoinAddress {
-        account: 0,
-        is_change: false,
-        address_index: 0,
+    let alice_path_to_address = HDAccountAddressId {
```

Comment:
I left this on purpose in case I wanted to try other addresses for the test, default is not even needed since the default will be used if no `path_to_address` was provided in the activation request. I will just use `HDAccountAddressId::default();` so that it's clear which address path is used.

Reply:
done